Description: Remove duplicate function name of formulas in ES translation
Author: Eike Rathke <er@openoffice.org>
Origin: upstream, http://www.openoffice.org/nonav/issues/showattachment.cgi/67975/i109407_BUSCAR.diff
Bug: http://qa.openoffice.org/issues/show_bug.cgi?id=109407
Forwarded: not-needed
Last-Update: 2010-02-22

--- /dev/null
+++ openoffice.org-3.2.0/ooo-build-3-2-0-6/patches/dev300/i109407_BUSCAR.diff
@@ -0,0 +1,20 @@
+--- OOO320/ooo.m12/l10n/source/es/localize.sdf	2009-12-14 15:38:04.000000000 +0100
++++ localize.sdf	2010-02-22 11:55:19.000000000 +0100
+@@ -3232,7 +3232,7 @@
+ formula	source\core\resource\core_resource.src	0	string	RID_STRLIST_FUNCTION_NAMES	SC_OPCODE_EXP_DIST			0	es	DISTR.EXP				2002-02-02 02:02:02
+ formula	source\core\resource\core_resource.src	0	string	RID_STRLIST_FUNCTION_NAMES	SC_OPCODE_FACT			0	es	FACT				2002-02-02 02:02:02
+ formula	source\core\resource\core_resource.src	0	string	RID_STRLIST_FUNCTION_NAMES	SC_OPCODE_FALSE			0	es	FALSO				2002-02-02 02:02:02
+-formula	source\core\resource\core_resource.src	0	string	RID_STRLIST_FUNCTION_NAMES	SC_OPCODE_FIND			0	es	BUSCAR				2002-02-02 02:02:02
++formula	source\core\resource\core_resource.src	0	string	RID_STRLIST_FUNCTION_NAMES	SC_OPCODE_FIND			0	es	ENCONTRAR				2002-02-02 02:02:02
+ formula	source\core\resource\core_resource.src	0	string	RID_STRLIST_FUNCTION_NAMES	SC_OPCODE_FISHER			0	es	FISHER				2002-02-02 02:02:02
+ formula	source\core\resource\core_resource.src	0	string	RID_STRLIST_FUNCTION_NAMES	SC_OPCODE_FISHER_INV			0	es	PRUEBA.FISHER.INV				2002-02-02 02:02:02
+ formula	source\core\resource\core_resource.src	0	string	RID_STRLIST_FUNCTION_NAMES	SC_OPCODE_FIXED			0	es	FIJO				2002-02-02 02:02:02
+@@ -3374,7 +3374,7 @@
+ formula	source\core\resource\core_resource.src	0	string	RID_STRLIST_FUNCTION_NAMES	SC_OPCODE_ROWS			0	es	FILAS				2002-02-02 02:02:02
+ formula	source\core\resource\core_resource.src	0	string	RID_STRLIST_FUNCTION_NAMES	SC_OPCODE_RSQ			0	es	COEFICIENTE.R2				2002-02-02 02:02:02
+ formula	source\core\resource\core_resource.src	0	string	RID_STRLIST_FUNCTION_NAMES	SC_OPCODE_SCHIEFE			0	es	COEFICIENTE.ASIMETRIA				2002-02-02 02:02:02
+-formula	source\core\resource\core_resource.src	0	string	RID_STRLIST_FUNCTION_NAMES	SC_OPCODE_SEARCH			0	es	BUSCAR				2002-02-02 02:02:02
++formula	source\core\resource\core_resource.src	0	string	RID_STRLIST_FUNCTION_NAMES	SC_OPCODE_SEARCH			0	es	HALLAR				2002-02-02 02:02:02
+ formula	source\core\resource\core_resource.src	0	string	RID_STRLIST_FUNCTION_NAMES	SC_OPCODE_SIN			0	es	SENO				2002-02-02 02:02:02
+ formula	source\core\resource\core_resource.src	0	string	RID_STRLIST_FUNCTION_NAMES	SC_OPCODE_SIN_HYP			0	es	SENOH				2002-02-02 02:02:02
+ formula	source\core\resource\core_resource.src	0	string	RID_STRLIST_FUNCTION_NAMES	SC_OPCODE_SLOPE			0	es	PENDIENTE				2002-02-02 02:02:02
--- openoffice.org-3.2.0.orig/ooo-build-3-2-0-6/patches/dev300/apply
+++ openoffice.org-3.2.0/ooo-build-3-2-0-6/patches/dev300/apply
@@ -2927,6 +2927,8 @@ sc-autofilter-l10n-es.diff, i#103840, pm
 # pack only selected languages in the l10n module
 l10n-build-only-selected-langs.diff, i#106185, pmladek
 
+i109407_BUSCAR.diff
+
 [ PieceBits ]
 # zipintro is not kind to us ...
 piece-desktop.diff
